cmd/timescale-prometheus/main.go

```diff
@@ -329,6 +340,54 @@ func parseFlags() (*config, error) {
 	return cfg, nil
 }
 
+var errTimedOut = fmt.Errorf("timed out waiting to connect to DB")
+
+func waitForDb(cfg *config) error {
+	maxTimeout := 60 * time.Second
+	addr := fmt.Sprintf("%s:%d", cfg.pgmodelCfg.Host, cfg.pgmodelCfg.Port)
```

Comment:
`var` declaration will be preferred in top of a function. It will save from allocation of any memory in runtime if the var is not used.